ARCs/arc-0069.md

```diff
@@ -0,0 +1,173 @@
+> * Adding the field `mime_type` to signal the MIME type and subtype of the media included in `au`. **OPTIONAL**
+> * Adding the field `media_url` which is used to link the high resolution media file. **OPTIONAL** but the sha-256 digest of the file **MUST** match the Asset Metadata Hash (`am`).
+
+
+The JSON Metadata schema is as follows:
+
+```json
+{
+    "title": "Token Metadata",
+    "type": "object",
+    "properties": {
+        "standard": {
+            "type": "string",
+            "description": "Describes the standard used, in this case arc69"
+        },
+        "description": {
+            "type": "string",
+            "description": "Describes the asset to which this token represents"
+        },
+        "external_url": {
+            "type": "string",
+            "description": "A URI pointing to an external website presenting the asset."
+        },
+        "media_url": {
+            "type": "string",
+            "description": "A URI pointing the high resolution media file of the asset."
+        },
+        "attributes": {
+            "type": "array",
+            "description": "Attributes following Open Sea's attributes format (https://docs.opensea.io/docs/metadata-standards#attributes)."
+        }
+    }
+}
+```
+The `standard` field is **REQUIRED**, all the other fields are **OPTIONAL**. But if provided, they **MUST** match the description in the JSON schema.
+
+The URI field (`external_url`) is defined similarly as the Asset URL parameter `au`.
+However, contrary to the Asset URL, it does not need to link to the digital media file.
+
+
+
+#### MIME Type
+
+In addition to including a MIME type in the `au` field of the asset, the JSON Metadata schema allows indication of the MIME type and subtype.
+
+
+
+#### Examples
+
+##### Basic Example
+
+An example of an ARC-69 JSON Metadata file for a song follows. The properties array proposes some **SUGGESTED** formatting for token-specific display properties and metadata.
+
+```json
+
+{   
+    "standard": "arc69",
+    "description": "arc69 theme song",
+    "external_url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+    "mime_type":"video/mp4",
+    "attributes": [
+        {"trait_type":"Bass",
+        "value":"Groovy"
+        },
+        {"trait_type":"Vibes",
+        "value":"Funky"
+        },
+        {"trait_type":"Overall",
+        "value":"Good stuff"
+        }
+    ]
+}
+```
+
+An example of possible ASA parameters would be:
+
+* *Asset Name*: `ARC69 theme song` for example.
+* *Unit Name*: `69TS` for example.
+* *Asset URL*: `ipfs://QmWS1VAdMD353A6SDk9wNyvkT14kyCiZrNDYAad4w1tKqT#v`
+* *Metadata Hash*: the 32 bytes of the SHA-256 digest of the high resolution media file.
+* *Total Number of Units*: 1
+* *Number of Digits after the Decimal Point*: 0
+
+#### Mutability
+
+##### Rendering
+
+Clients SHOULD render an ASA's latest ARC69 metadata. Clients MAY render an ASA's previous ARC69 metadata for changelogs or other historical features.

```

Comment:
```suggestion
Clients **SHOULD** render an ASA's latest ARC69 metadata. Clients **MAY** render an ASA's previous ARC69 metadata for changelogs or other historical features.

Clients **SHOULD** indicate whether the ARC69 metadata is mutable or not (see below).
```

Reply:
> Clients **SHOULD** indicate whether the ARC69 metadata is mutable or not (see below).

I'd prefer to leave this up to clients. Currently none of them indicate whether ARC69 metadata is mutable, and that seems like a product decision

Reply:
ok, let's not add this